.azure-pipelines/test-eh-sb-sdk.yml

```diff
@@ -35,11 +35,10 @@ steps:
   - script: |
       python --version
       python -m pip install pytest
-      echo "Installing $DownloadArtifactWhl"
+      echo "Installing $(DownloadArtifactWhl)"
       python -m pip install $(Pipeline.Workspace)/$(DownloadArtifactFolder)/$(DownloadArtifactWhl)
       cd $(SdkPath)
       python -m pip install futures
-      python -m pip install --ignore-installed .
```

Comment:
I think you need this line to be python -m pip install .

Reply:
whoops yes, will fix